hbase-oss/pom.xml

```diff
@@ -70,56 +71,206 @@
           </systemProperties>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>aggregate-into-a-jar-with-relocated-third-parties</id>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <createDependencyReducedPom>true</createDependencyReducedPom>
```

Comment:
Please also configure `dependencyReducedPomLocation` as per https://maven.apache.org/plugins/maven-shade-plugin/shade-mojo.html#dependencyReducedPomLocation to move it inside of `${project.build.dir}` (more commonly known as:`target/`)

Reply:
setting the location doesn't actually work. it's a long standing problem with the shade plugin that keeps getting only half fixed.

this is why we don't change it's location in the main project repo's shaded jar work. (or over in the hadoop project either)

Reply:
Really? I wonder how I've thought this was working elsewhere then.. Strange.

What about copying the clean-plugin configuration to get rid of it and updating .gitignore then? (as a follow on)